pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -435,30 +444,73 @@ cdef class {{name}}HashTable(HashTable):
         }
 
     cpdef get_item(self, {{dtype}}_t val):
+        """Extracts the position of val from the hashtable.
```

Comment:
can the return be typed as int Py_ssize_t or something? same question for get_na below